Tools/freeze/freeze.py

```diff
@@ -217,7 +217,12 @@ def main():
 
     # locations derived from options
     version = '%d.%d' % sys.version_info[:2]
-    flagged_version = version + sys.abiflags
+    if not win:
```

Comment:
I would prefer to test hasattr(sys, 'abiflags') rather than "not win".

Reply:
Yeah, that would work too.